### PR TITLE
refactor: improve code formatting and quality.

### PR DESCRIPTION
**Pull request by Akaikenlol**

### CODE REVIEW

File: src/components/common/WebsiteCard.tsx [hunk 1]
```diff
@@ -1,14 +1,6 @@
...
 import TagBar from "./TagBar";
-
-type WebsiteCardProps = {
-  id: string;
-  title: string;
-  tags: string[];
-  primaryAuthor: string;
-  coverImage: string;
-  profile: string;
-};
```
Comment (Rickaym) [inline src/components/common/WebsiteCard.tsx:11]: Why was this removed?

Comment (Rickaym) [reply src/components/common/WebsiteCard.tsx:11]: Ah, nvm i just realized it was only moved

Comment (Akaikenlol) [reply src/components/common/WebsiteCard.tsx:11]: yeah, I adjust it.

**Review by Rickaym — APPROVED** [review]
LGTM
